nltk/stem/lancaster.py

```diff
@@ -304,7 +319,14 @@ def __applyRule(self, word, remove_total, append_string):
             word += append_string
         return word
 
+    def strip_prefix(self, word):
+        """Remove prefix from a word.
+        """
```

Comment:
This is interesting, it seems like many versions of Porter stemmer (except NLTK's) contains that function ;P

https://www.google.com.sg/search?client=safari&rls=en&q=for+prefix+in+(%22kilo%22,+%22micro%22,+%22milli%22,+%22intra%22,+%22ultra%22,+%22mega%22